### PR TITLE
Added parameter to monitor a specific jail only

### PR DESCRIPTION
**Pull request by netson**
This has been available in the original check_command for years but hasn't made it into the checkcommand yet. This replaces my original PR, as that contained an error and also lacked updated documentation (my apologies).

Original PR: https://github.com/Icinga/icinga2/pull/7930

### CODE REVIEW

File: doc/10-icinga-template-library.md [hunk 1]
```diff
@@ -3304,7 +3304,8 @@ fail2ban_path           | **Optional.** Specify the path to the tw_cli binary, d
 fail2ban_warning        | **Optional.** Specify a warning threshold, default is 1
 fail2ban_critical       | **Optional.** Specify a critical threshold, default is 2
 fail2ban_socket         | **Optional.** Specify a socket path, default is unset
-fail2ban_perfdata       | **Optional.** If set to true, activate the perfdata output, default value for the plugin is set to true.
+fail2ban_perfdata       | **Optional.** If set to true, activate the perfdata output, default value for the plugin is set to true
+fail2ban_jail           | **Optional.** Specify the name of the specific jail to monitor; default is to omit and thus monitor all jails.
```
Comment (Al2Klimov) [inline doc/10-icinga-template-library.md:3308]: Why did you remove the period on the one line (right as conforms to the others), but put one at the end of the other line (wrong)?

File: itl/plugins-contrib.d/network-services.conf [hunk 2]
```diff
@@ -113,6 +113,10 @@ object CheckCommand "fail2ban" {
 			set_if = "$fail2ban_perfdata$"
 			description = "If set to true, activate the perfdata output"
 		}
+		"-j" = {
+			value = "$fail2ban_jail$"
+			description = "Specify the name of the specific jail to monitor; default is to omit and thus monitor all jails"
```
Comment (Al2Klimov) [inline itl/plugins-contrib.d/network-services.conf:118]: I'd say "omitted by default, i.e. all jails are being monitored"

File: doc/10-icinga-template-library.md (hunk 1 above)
Comment (netson) [reply doc/10-icinga-template-library.md:3308]: Because it seems linguistically correct: since it's a listing, we end the last one with a period. So I simply moved the period to the last item in the list.

File: itl/plugins-contrib.d/network-services.conf (hunk 2 above)
Comment (netson) [reply itl/plugins-contrib.d/network-services.conf]: fixed in updated commit

File: doc/10-icinga-template-library.md (hunk 1 above)
Comment (Al2Klimov) [inline doc/10-icinga-template-library.md:3308]: Please make this description the same as the one in the config file.

Comment (netson) [reply doc/10-icinga-template-library.md:3308]: There doesn't seem to be one? The -j parameter is not listed when you run check_fail2ban --help; it is however described in the documentation. See also: https://github.com/fail2ban/fail2ban/blob/master/files/nagios/check_fail2ban

Comment (netson) [reply doc/10-icinga-template-library.md:3308]: Ah sorry, I misunderstood; fix coming right up

Comment (Al2Klimov) [reply doc/10-icinga-template-library.md:3308]: No, I mean you just adjusted the text (as I suggested) in the config file, but not in the docs.

Comment (netson) [reply doc/10-icinga-template-library.md]: Fixed; sorry for the misunderstanding, sometimes I try to do things too fast! :-)

**Comment by netson** [comment]
> Now please (finally) squash you commits, i.e.:

Done.

**Comment by netson** [comment]
FYI, I also submitted a PR to the fail2ban project to add the jail parameter to the check_fail2ban help message and documentation (https://github.com/fail2ban/fail2ban/pull/2676)

**Review by julianbrost — APPROVED** [review]
